daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1584,35 +1584,35 @@ object Main extends Logging {
     } catch {
       case s: scala.util.control.ControlThrowable => throw s
       case e: java.io.FileNotFoundException => {
-        log(LogLevel.Error, "%s", e.getMessage())
+        log(LogLevel.Error, "Missing File Error", e.getMessage())
```

Comment:
You need tests that elicit these messages. I think they would show you that you need "Missing File Error: %s" otherwise there is no place for the e.getMessage() to get substituted into.

But I also think you should compute the message string from the exception object class name. 

So like this:
```
log(LogLevel.Error, "%s %s", Misc.getNameFromClass(e), e.getMessage())
```
The Misc.getNameFromClass(e) is a utility we created for exactly this sort of purpose. It gives you the class name without the package part.

Reply:
I imagine in this case and the ones below, the message from the exception is what we want to display. This new strings dont' provide enough information. For example, the message I get when we hit this FileNotFoundException is 

> [error] notafile.txt (No such file or directory)

Which is maybe a bit terse, but I think provides all the necessary information. We probably want CLI tests to make sure we are getting reasonable error messages.